TestApp/Sources/Data/HighlightModel.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright 2021 Readium Foundation. All rights reserved.
+//  Use of this source code is governed by the BSD-style license
+//  available in the top-level LICENSE file of the project.
+//
+
+import Combine
+import Foundation
+import GRDB
+import R2Shared
+import R2Navigator
+import UIKit
+
+struct HighlightRecord: Codable {
```

Comment:
Since we will remove the `Highlight` model from the Navigator, you can rename this one.

```suggestion
struct Highlight: Codable {
```